VimeoNetworkingExample-iOS/VimeoNetworkingCommonTests/RequestTests.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  RequestTests.swift
+//  VimeoNetworkingExample-iOSTests, VimeoNetworkingExample-tvOSTests
+//
+//  Created by Westendorf, Mike on 5/21/17.
+//  Copyright © 2016 Vimeo. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import XCTest
+@testable import VimeoNetworking
+
+class RequestTests: XCTestCase
+{
+    let retryPolicyTestClosure = { (policyToTest: RetryPolicy, comparePolicy: RetryPolicy) -> Bool in
+        switch policyToTest
+        {
+        case .singleAttempt:
+            switch comparePolicy
+            {
+            case .singleAttempt:
+                return true
+            default:
+                return false
+            }
+        default:
+            if case .multipleAttempts(let attemptCount, let initialDelay) = policyToTest,
+                case .multipleAttempts(let testValueCount, let testDelay) = comparePolicy
+            {
+                return attemptCount == testValueCount && initialDelay == testDelay
+            }
+            
+            return false
+        }
+    }
+    
+    func test_CacheFetchPolicy_DefaultPolicyReturnsNetworkOnly()
```

Comment:
That's basically the naming convention that this is following, the thing being tested is the CacheFetchPolicy.  There isn't really a condition, the default policy for all methods should return .networkOnly, so the name test_CacheFetchPolicy_DefaultPolicyReturnsReturnsNetworkOnly seemed to properly separate what's being tested and the expected outcome.